### ISSUE TITLE
Focus indicator on Expand All+ is partially obscured by top of the review accordion

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#95527]
## Issue Description
This issue was identified in the [accessibility audit of the review & confirmation pages ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/94614)of the 526 form.

The bottom of the yellow focus indicator is covered  by the top of the accordion. Note: this is a WCAG 2.2 Level AAA, so not required, more of an enhancement.

![Image](https://github.com/user-attachments/assets/bac1f496-bc9c-479d-920f-951c8df8af06)

[**Expected severity**](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric): a11y-defect-4
**Page**: review-and-submit
**Environment**: Keyboard Nav
**Estimated Level of Effort to fix**: low
**Design system component involved**: Yes
**Relevant WCAG success criteria**: [2.4.12 Focus Not Obscured (Enhanced) Level AAA](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview&showtechniques=413,2412#focus-not-obscured-enhanced)

### To Reproduce:
- TAB to the "Expand all +" button

#### Expected Behavior
-  The keyboard focus indicator should be fully visible

#### Actual Behavior
- The bottom of the focus ring is hidden

## Tasks
- [ ] Adjust styling so that the focus ring is fully visible
- [ ] Please update the "Fix Status" column in the [526 Accessibility Audit issues tracking doc](https://docs.google.com/spreadsheets/d/1paNxD798NSMrjZeUe9WwVQ4xRdl439ypKyPDPW3CXIo/edit?gid=2034611899#gid=2034611899) when ticket is started and completed.

## Acceptance Criteria
- [ ] The keyboard focus indicator should be fully visible
- [ ] Accessibility Audit issue tracking doc is updated with ticket status
